code/datums/martial/boxing.dm

```diff
@@ -2,23 +2,26 @@
 	name = "Boxing"
 	id = MARTIALART_BOXING
 
-/datum/martial_art/boxing/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+/datum/martial_art/boxing/disarm_act(mob/living/A, mob/living/D)
 	to_chat(A, "<span class='warning'>Can't disarm while boxing!</span>")
-	return 1
+	return TRUE
 
-/datum/martial_art/boxing/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+/datum/martial_art/boxing/grab_act(mob/living/A, mob/living/D)
 	to_chat(A, "<span class='warning'>Can't grab while boxing!</span>")
-	return 1
+	return TRUE
+
+/datum/martial_art/boxing/harm_act(mob/living/A, mob/living/D)
 
-/datum/martial_art/boxing/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	var/mob/living/carbon/human/attacker_human = A
+	var/datum/species/species = attacker_human.dna.species
 
 	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
 
 	var/atk_verb = pick("left hook","right hook","straight punch")
 
-	var/damage = rand(5, 8) + A.dna.species.punchdamagelow
+	var/damage = rand(5, 8) + species.punchdamagelow
```

Comment:
would it be possible to let nonhumanoids use melee_attack_lower instead of punchdamagelow (with humanoids still using punchdamagelow, of course) for the purposes of boxing so that we can remove the "humanoids only" restriction of boxing?

Reply:
You have to be able to wear gloves anyway

Reply:
yeah, but what if an admin makes a boxing monkey or something? plus, the premise of this PR is allowing monkeys to use martial arts, no?

Reply:
OK so how do I deal with miss, attack sounds and punchdamage?